Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageRepliesView.swift

```diff
@@ -116,3 +116,42 @@ public struct MessageRepliesView<Factory: ViewFactory>: View {
         }
     }
 }
+
+/// Lazy view that uses the message controller to fetch the parent message before creating message replies view.
+/// This is needed when the parent message is not available in the local cache.
+/// Changing the `parentMessage` to `nil` in the `MessageRepliesView` would case multiple changes including breaking changes.
+struct LazyMessageRepliesView<Factory: ViewFactory>: View {
+    @State private var parentMessageObserver: ChatMessageController.ObservableObject
+
+    var factory: Factory
+    var channel: ChatChannel
+    var message: ChatMessage
+
+    init(
+        factory: Factory,
+        channel: ChatChannel,
+        message: ChatMessage,
+        parentMessageController: ChatMessageController
+    ) {
+        parentMessageObserver = parentMessageController.observableObject
+        self.factory = factory
+        self.channel = channel
+        self.message = message
+        parentMessageObserver.controller.synchronize()
```

Comment:
shouldn't we do this only `onAppear`? `init` can be called many times.

Reply:
Xcode 15 was not happy with the current implementation, so I moved to onAppear, but this requires changing from Group, to use a VStack